packages/nextjs/scaffold.config.ts

```diff
@@ -19,6 +33,9 @@ const scaffoldConfig = {
   // It's recommended to store it in an env variable:
   // .env.local for local testing, and in the Vercel/system env config for live apps.
   alchemyApiKey: process.env.NEXT_PUBLIC_ALCHEMY_API_KEY || "oKxs-03sij-U_N0iOlrSsZFr29-IqbuF",
+
+  // This is the contract data that was generated by "yarn deploy", or null if no local contracts are used
+  contracts,
```

Comment:
When setting this to `null`, I get the following type error in the project:

![image](https://user-images.githubusercontent.com/2486142/227499236-e42fc606-4d0c-4368-b0ac-4ba1c7d24ac6.png)

Reply:
Hmmm, strange that this didn't fail before. I do see it now, though. Fixed it :+1: